src/Shared/CommunicationsUtilities.cs

```diff
@@ -315,7 +315,7 @@ private static long GenerateHostHandshakeFromBase(long baseHandshake)
             // Mask out the first byte. That's because old
             // builds used a single, non zero initial byte,
             // and we don't want to risk communicating with them
-            return baseHandshake;
+            return baseHandshake & 0x00FFFFFFFFFFFFFF;
```

Comment:
Cross-link to the code that sends/rejects the 5f 60 bytes, and make sure to mention https://github.com/dotnet/runtime/issues/37333 in your commit message.

Reply:
To be clear, do you mean get a permalink to that code bit when I squash it or put it as a comment in the code?

Reply:
comment in the code for the other code that checks. and maybe remove the "old builds" reference. link to the bug in the commit message or PR description, not in code.